R/rdeephaven/R/aggregate_wrapper.R

```diff
@@ -1,95 +1,85 @@
     }
   )
 )
 
 ### All of the functions below return an instance of the above class
 
-#' @export
 agg_first <- function(cols = character()) {
   verify_string("cols", cols, FALSE)
-  return(Aggregation$new(INTERNAL_agg_first(cols)))
+  return(Aggregation$new(INTERNAL_agg_first, "agg_first", cols=cols))
 }
 
-#' @export
 agg_last <- function(cols = character()) {
   verify_string("cols", cols, FALSE)
-  return(Aggregation$new(INTERNAL_agg_last(cols)))
+  return(Aggregation$new(INTERNAL_agg_last, "agg_last", cols=cols))
 }
 
-#' @export
 agg_min <- function(cols = character()) {
   verify_string("cols", cols, FALSE)
-  return(Aggregation$new(INTERNAL_agg_min(cols)))
+  return(Aggregation$new(INTERNAL_agg_min, "agg_min", cols=cols))
 }
 
-#' @export
 agg_max <- function(cols = character()) {
   verify_string("cols", cols, FALSE)
-  return(Aggregation$new(INTERNAL_agg_max(cols)))
+  return(Aggregation$new(INTERNAL_agg_max, "agg_max", cols=cols))
 }
 
-#' @export
 agg_sum <- function(cols = character()) {
   verify_string("cols", cols, FALSE)
-  return(Aggregation$new(INTERNAL_agg_sum(cols)))
+  return(Aggregation$new(INTERNAL_agg_sum, "agg_sum", cols=cols))
 }
 
-#' @export
 agg_abs_sum <- function(cols = character()) {
   verify_string("cols", cols, FALSE)
-  return(Aggregation$new(INTERNAL_agg_abs_sum(cols)))
+  return(Aggregation$new(INTERNAL_agg_abs_sum, "agg_abs_sum", cols=cols))
 }
 
-#' @export
 agg_avg <- function(cols = character()) {
   verify_string("cols", cols, FALSE)
-  return(Aggregation$new(INTERNAL_agg_avg(cols)))
+  return(Aggregation$new(INTERNAL_agg_avg, "agg_avg", cols=cols))
 }
 
-#' @export
 agg_w_avg <- function(wcol, cols = character()) {
   verify_string("wcol", wcol, TRUE)
   verify_string("cols", cols, FALSE)
-  return(Aggregation$new(INTERNAL_agg_w_avg(wcol, cols)))
+  return(Aggregation$new(INTERNAL_agg_w_avg, "agg_w_avg", wcol=wcol, cols=cols))
 }
 
-#' @export
 agg_median <- function(cols = character()) {
   verify_string("cols", cols, FALSE)
-  return(Aggregation$new(INTERNAL_agg_median(cols)))
+  return(Aggregation$new(INTERNAL_agg_median, "agg_median", cols=cols))
 }
 
-#' @export
 agg_var <- function(cols = character()) {
   verify_string("cols", cols, FALSE)
-  return(Aggregation$new(INTERNAL_agg_var(cols)))
+  return(Aggregation$new(INTERNAL_agg_var, "agg_var", cols=cols))
 }
 
-#' @export
 agg_std <- function(cols = character()) {
   verify_string("cols", cols, FALSE)
-  return(Aggregation$new(INTERNAL_agg_std(cols)))
+  return(Aggregation$new(INTERNAL_agg_std, "agg_std", cols=cols))
 }
 
-#' @export
 agg_percentile <- function(percentile, cols = character()) {
   verify_in_unit_interval("percentile", percentile, TRUE)
   verify_string("cols", cols, FALSE)
-  return(Aggregation$new(INTERNAL_agg_percentile(percentile, cols)))
+  return(Aggregation$new(INTERNAL_agg_percentile, "agg_percentile", percentile=percentile, cols=cols))
 }
 
-#' @export
-agg_count <- function(col) {
+agg_count <- function(col = "n") {
```

Comment:
1) Do we have this default in other clients?
2) We endorse camel case with starting caps for column names.  This does not match that convention.

Reply:
This default does not exist in other clients. I will remove it here.